docs/src/upgrade/release-3.2.x-incubating.asciidoc

```diff
@@ -88,6 +88,11 @@ to the list of locally computed clauses.
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-1764[TINKERPOP-1764]
 
+Clone a graph
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
These `^` need to be of the same length as the preceding title line otherwise it won't render correctly.

Reply:
fixed